### PR TITLE
Migrate `ShardChainsList.tsx` to Chakra

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
Migrate `ShardChainsList.tsx` to Chakra

## Related Issue
#8632

### CODE REVIEW

File: src/components/ShardChainsList.tsx [hunk 1]
```diff
@@ -102,9 +98,9 @@ const ShardChainsList: React.FC<IProps> = () => {
   ]
 
   return (
-    <Container>
+    <Box as="div" mb="4rem">
```
Comment (TylerAPfledderer) [inline src/components/ShardChainsList.tsx:101]: Same as the `MergeArticleList` migration PR, `Box` renders `div` by default.
```suggestion
    <Box mb="4rem">
```

File: src/components/ShardChainsList.tsx [hunk 2]
```diff
@@ -1,14 +1,10 @@
 import React from "react"
-import styled from "@emotion/styled"
-import CardList, { CardListItem } from "./CardList"
 import { useIntl } from "react-intl"
+import CardList, { CardListItem } from "./CardList"
 
+import { Box } from "@chakra-ui/react"
```
Comment (TylerAPfledderer) [inline src/components/ShardChainsList.tsx:5]: Import order

**Review by TylerAPfledderer — CHANGES_REQUESTED** [review]
Hi @MahendraBishnoi29! Tiny changes here, then it should be good!

**Comment by MahendraBishnoi29** [comment]
> Hi @MahendraBishnoi29! Tiny changes here, then it should be good!

thank you for fixing my small mistakes @TylerAPfledderer 🚀

**Review by pettinarip — APPROVED** [review]
🚀 🚀 🚀 🚀

**Pull request merged**